java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/AbstractPlugin.java

```diff
@@ -17,21 +17,15 @@
 import org.eclipse.keyple.core.seproxy.ReaderPlugin;
 import org.eclipse.keyple.core.seproxy.SeReader;
 import org.eclipse.keyple.core.seproxy.event.ObservablePlugin;
-import org.eclipse.keyple.core.seproxy.event.PluginEvent;
 import org.eclipse.keyple.core.seproxy.exception.*;
 import org.eclipse.keyple.core.util.Configurable;
 import org.eclipse.keyple.core.util.Nameable;
-import org.eclipse.keyple.core.util.Observable;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 /**
  * Observable plugin. These plugin can report when a reader is added or removed.
  */
-public abstract class AbstractPlugin extends Observable<PluginEvent>
-        implements ReaderPlugin, Nameable, Configurable {
-
-    private static final Logger logger = LoggerFactory.getLogger(AbstractPlugin.class);
+public abstract class AbstractPlugin
+        implements ReaderPlugin, ObservablePlugin, Nameable, Configurable {
```

Comment:
AbstractPlugin shouldn't implements ObservablePlugin
AbstractPlugin shouldn't implements 'explicitely' Nameable & Configurable, instead should extends AbstractSeProxyComponent